packages/vuetify/src/components/VForm/VForm.ts

```diff
@@ -94,19 +97,19 @@ export default {
         }, 0)
```

Comment:
return types might be useful for the public methods? so `bool` for `validate()` and `void` for the others?